packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -237,6 +247,8 @@ export function captureObserverEntries(list, { capturePaint }) {
   }
 
   calculateTotalBlockingTime(longtaskEntries)
+  const clsEntries = list.getEntriesByType(LAYOUT_SHIFT)
```

Comment:
Please add some doc comments upfront and also link to spec. Helps a lot while going over the code.